src/charm.py

```diff
@@ -1182,8 +1182,9 @@ def _on_image_relation_joined(self, _: ops.RelationJoinedEvent) -> None:
             return
 
         clouds_yaml = state.charm_config.openstack_clouds_yaml
-        cloud = list(clouds_yaml["clouds"].keys())[0]
-        auth_map = clouds_yaml["clouds"][cloud]["auth"]
+        # unsubscriptable-object / E1136 thinks cloud_yaml is kind of a list
+        cloud = list(clouds_yaml["clouds"].keys())[0]  # pylint: disable=E1136
+        auth_map = clouds_yaml["clouds"][cloud]["auth"]  # pylint: disable=E1136
```

Comment:
Usually we put the pylint rule name rather than the ID to make it easier to undersantd. I realise it is above, just better not to use the ID at all